cocnept-designs/basic.css

```diff
@@ -6,6 +6,17 @@ body {
     margin: 0;
 }
 
+/* todo: extend to other browsers */
+*::-webkit-scrollbar {
+    width: 5px;
+}
+*::-webkit-scrollbar-thumb {
+    background-color: #333;
+}
+*::-webkit-scrollbar-track {
```

Comment:
I don't think `*` is necessary lol; also, the scrollbar is already hidden on like every OS except Windows; maybe we could detect if the scrollbar takes up space and apply the scrollbar styling accordingly? (Because I'm assuming the scrollbar styling is only to hide the ugly default one on Windows)

Reply:
Okay, but "every OS except Windows" is basically Apple so.
(Linux and Chrome don't exist!!!!!)
Do we get notifications for these comments anyway?

Reply:
uh @Nichodon idk but I do; it just doesn't show why I got the notif very clearly